psi4/src/psi4/dfocc/ccd_W_intr.cc

```diff
@@ -154,6 +154,124 @@ void DFOCC::ccd_WmnijT2() {
 
 }  // end ccd_WmnijT2
 
+//======================================================================
+//    WmnijT2AA
+//======================================================================
+void DFOCC::ccd_W_MNIJT2AA()
+{
+    SharedTensor2d J, W, I, X, Y, T;
+    SharedTensor2d T2, Tau, T2new;
+
+    // W_MNIJ Alpha Block
+    // W_MNIJ = <MN||IJ> + \sum_(EF) T(IJ,EF) * <MN|EF>
```

Comment:
Once again, the new code includes a comment defining the quantity in full, but the older restricted code includes a comment describing each line. (In other words, where is the restricted code's counterpart to l. 165? I can find its counterpart to 166.)

@bozkaya, how much of this is deliberate? Is this a new code style that you're introducing, but not wanting to change _all_ the code right away?